mobile/native/src/ln_dlc/mod.rs

```diff
@@ -512,9 +512,10 @@ fn keep_wallet_balance_and_history_up_to_date(node: &Node) -> Result<()> {
                     status,
                     wallet_type: WalletHistoryItemType::DlcChannelFunding {
                         funding_txid: details.txid.to_string(),
-                        // this is not 100% correct as fees are not exactly divided by 2. The fee a
-                        // user has to pay depends on his final address.
-                        reserved_fee_sats: details.fee.map(|fee| fee / 2),
+                        // this is not 100% correct as fees are not exactly divided by 2. The share
```

Comment:
Maybe we should just fix this in rust-dlc and say the fee is always 50:50

Reply:
Hmm, I don't think it should be fixed there, because it wouldn't make sense to me to charge the user more just because we as the coordinator bring 7 inputs or something.

I think it would be nicer to record all the fee contributions in a channel metadata table. I think this is something we should be able to achieve after the new model is introduced.